test/integration/cart_integration_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class CartIntegrationTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test 'an unauthorized user can add an item to the cart' do
+    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
```

Comment:
Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.